src/Rules.Framework.Providers.MongoDb/MongoDbProviderRulesDataSource.cs

```diff
@@ -115,6 +116,10 @@ private static FilterDefinition<RuleDataModel> BuildFilterByContentTypeAndDatesI
             var contentTypeFilter = Builders<RuleDataModel>.Filter.Eq(x => x.ContentType, contentType.ToString());
 
             var datesFilter = Builders<RuleDataModel>.Filter.And(
+                Builders<RuleDataModel>.Filter.Or(
+                    Builders<RuleDataModel>.Filter.Eq(rule => rule.Active, null),
+                    Builders<RuleDataModel>.Filter.Eq(rule => rule.Active, true)
+                ),
```

Comment:
I also didn't catch this on the review, I think you should remove these filters by "active". For the same reason as we talked about on the `InMemoryProviderRulesDataSource`.